website/docs/docs/develop/run-your-dbt-projects.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Run your dbt projects"
+id: "run-your-dbt-projects"
+---
+You can run your dbt projects with [dbt Cloud](/docs/develop/dbt-cloud-features) and [dbt Core](https://github.com/dbt-labs/dbt-core). dbt Cloud is a hosted application where you can develop directly from a web browser. dbt Core is an open source project where you can develop from the command line.
+
+Among other features, dbt Cloud provides a hosted development environment to help you build, test, run, and version control your project faster. It also includes a purpose-built scheduler and an easier way to share your [dbt project's documentation](/docs/collaborate/build-and-view-your-docs) with your team. These development tasks are directly built into dbt Cloud. See [dbt Cloud IDE] (link to one of the IDE pages mirna is working on) for more details.
+
+With dbt Core, you can run your dbt projects from the command line. The command line interface (CLI) is available from your computer's terminal application such as Terminal and iTerm. When using the command line, you can run commands and do other work from the current working directory on your computer. Before running the dbt project from the command line, make sure you are working in your dbt project directory. Learning terminal commands such as `cd` (change directory), `ls` (list directory contents), and `pwd` (present working directory) can help you navigate the directory structure on your system.
+
+When working with a project, the dbt commands you commonly use are:
```

Comment:
i'm suggesting being explicit here with the cli and ide

```suggestion
When working with a project in the CLI or Cloud IDE, the dbt commands you commonly use are:
```

Reply:
i don't think the sentence is confusing or ambiguous without "in the CLI or Cloud IDE" so wanna lean toward being concise (fewer words). 
 
do you find it unclear?

Reply:
oh yea i did initially - i think it's because it followed the dbt core summary maybe? so i had to re read to make sure the commands were related to both